cyclonedx/model/component.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, name: str, component_type: ComponentType = ComponentType.LIBR
                  external_references: Optional[List[ExternalReference]] = None,
                  properties: Optional[List[Property]] = None, release_notes: Optional[ReleaseNotes] = None,
                  # Deprecated parameters kept for backwards compatibility
-                 namespace: Optional[str] = None, license_str: Optional[str] = None
+                 namespace: Optional[str] = None, license_str: Optional[str] = None, cpe: Optional[str] = None
```

Comment:
@jblu42  i see the confusion. 
please put the new parameter in a new line above the `# Deprecated parameters kept for backwards compatibility` 
just like done here: https://github.com/CycloneDX/cyclonedx-python-lib/pull/137/files#diff-b1cd177706b0ed9acbe48de561ac6f96186fc1a291b00e757b30b514fd3acadaR110